<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaCadastroLogin</class>
 <widget class="QMainWindow" name="TelaCadastroLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Usuário</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../Imagens/icone.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="Cadastrar">
   <widget class="QPushButton" name="PlanodeFundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-4</y>
      <width>871</width>
      <height>519</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/grr.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Cadastro">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>98</y>
      <width>169</width>
      <height>125</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Imagens/IconeCadastro.png</normaloff>
      <disabledoff>../Imagens/IconeCadastro.png</disabledoff>
      <disabledon>../Imagens/IconeCadastro.png</disabledon>../Imagens/IconeCadastro.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>0</number>
    </property>
    <property name="autoRepeatInterval">
     <number>0</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cadastro">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>165</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_cadastro.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="IconeSair">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <disabledon>../Imagens/Icone sair.png</disabledon>
      <selectedon>../Imagens/Icone sair.png</selectedon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>384</y>
      <width>173</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 12pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>*Senha mestre incorreta*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Caixa_Senha">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>282</y>
      <width>375</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 233);
font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Caixa_SuperUser">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>336</y>
      <width>375</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 233);
font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Super Mestre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Caixa_Usuario">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>375</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 233);
font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Usuário</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
